docs/2.7.0/docs/daml-finance/tutorials/getting-started/lifecycling.rst

```diff
@@ -144,6 +144,28 @@ process up to the implementation.
 * The party executing settlement can be chosen as well, as described in the previous tutorial on
   :doc:`Settlement <settlement>`.
 
+Which party should take the role as *lifecycler*?
+=================================================
+
+From a design perspective, a lifecycler is often the party that defines the lifecycle events
+happening on an instrument (although they can be different). In the simplified example above, it is
+the bank. In a more realistic example, it would probably be the issuer.
+In some special cases like Elections, if we really need the owner to be the lifecycler in order to
+choose which "path" they'd like to lifecycle the instrument with, we can use a delegation contract.
```

Comment:
```suggestion
choose which "path" they would like to lifecycle the instrument with, we can use a delegation contract.
```

Reply:
I would remove the part on elections as it' quite out of context. At this stage the reader hasn't even seen an example with elections

Reply:
I took out that part now, and kept the part with "some special cases ... delegation contract"